src/Command/CommandBase.php

```diff
@@ -1517,6 +1517,10 @@ private function promptChooseAcsfSite(EnvironmentResponse $cloudEnvironment): mi
         foreach ($acsfSites['sites'] as $domain => $acsfSite) {
             $choices[] = "{$acsfSite['name']} ($domain)";
         }
+        if (!count($choices)) {
+            throw new AcquiaCliException("Could not get ACSF sites");
```

Comment:
You'll need to add a test case to cover this line. I'd suggest copying testRefreshAcsfFiles from PullFilesCommandTest.php and modify it to expect this exception. You'll need to modify mockGetAcsfSites() to return 0 sites to trigger the exception.